src/OpenTelemetry.Api/OpenTelemetry.Api.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
```

Comment:
Is the change only for ordering? Does VSCode enforce the ordering of `TargetFrameworks` if so?

Reply:
Unfortunately, you are correct. VS Code C# extension does not work good with projects that target multiple frameworks.
More:
- https://github.com/OmniSharp/omnisharp-vscode/issues/1783
- https://github.com/OmniSharp/omnisharp-vscode/issues/3754
- https://cloud-native.slack.com/archives/C01N3BC2W7Q/p1617727862056600